src/services/routing/pendle.ts

```diff
@@ -196,17 +171,19 @@ const getPendleSwapExactTokenForPtQuote = async (
       underlyingAmountInForPt = underlyingSwapData.minAmountOut;
     }
 
-    const staticRouter = getPendleStaticRouterContract();
-    const pendleSlippage = DEX_SLIPPAGE_BPS * 100000000000000n;
-    const result = await staticRouter.read.swapExactTokenForPtStaticAndGenerateApproxParams([
```

Comment:
this func throws an error "Slippage: APPROX_EXHAUSTED" because it performs a binary search to approximate the quote using some default params https://etherscan.io/address/0x263833d47eA3fA4a30f269323aba6a107f9eB14C#readProxyContract `getDefaultApproxParams` which searches between max uint256 and 0.

in my tests it threw it quite often, but I was testing with small amounts which may be why it threw more often than not. either way, these new approach mirrors our approach for the other swap (`getPendleSwapExactPtForTokenQuote`) and seems to work well.